packages/hping3/hping3.patch

```diff
@@ -0,0 +1,326 @@
+ #else
+ # error can not find the byte order for this architecture, fix bytesex.h
+diff -Naur hping3-20051105/configure hping3-3.0.0/configure
+--- hping3-20051105/configure	2004-06-04 12:09:10.000000000 +0530
++++ hping3-3.0.0/configure	2018-02-22 16:27:46.000000000 +0530
+@@ -1,4 +1,4 @@
+-#!/bin/sh
++#!/data/data/com.termux/files/usr/bin/sh
+ 
+ show_help()
+ {
+@@ -15,14 +15,14 @@
+ 
+ CC=${CC:=cc}
+ 
+-echo build byteorder.c...
+-$CC byteorder.c -o byteorder || exit 1
++#echo build byteorder.c...
++#$CC byteorder.c -o byteorder || exit 1
+ 
+ INSTALL_MANPATH=`echo $MANPATH|cut -f1 -d:`
+ if [ "$INSTALL_MANPATH" = "" ]; then
+-	INSTALL_MANPATH="/usr/local/man"
++	INSTALL_MANPATH="$PREFIX/share/man"
+ fi
+-BYTEORDER=`./byteorder -m`
++BYTEORDER=__LITTLE_ENDIAN_BITFIELD
+ 
+ echo create byteorder.h...
+ cat > byteorder.h <<EOF
+@@ -61,52 +61,52 @@
+ #
+ # TCL detection
+ #
+-for TCLPATH_TRY in "/usr/bin/" "/usr/local/bin/" "/bin/"
+-do
+-	for TCLVER_TRY in "8.4" "8.3" "8.2" "8.1" "8.0"
+-	do
+-		if [ -z $TCLSH ]
+-		then
+-			TCLSH_TRY=${TCLPATH_TRY}tclsh${TCLVER_TRY}
+-			if [ -f $TCLSH_TRY ]
+-			then
+-				TCLSH=$TCLSH_TRY
+-				echo "===> Found Tclsh in: $TCLSH"
+-			fi
+-		fi
+-	done
+-done
+-if [ -f $TCLSH ]
+-then
+-	TCL_VER=`echo puts \\$tcl_version | $TCLSH -`
+-	USE_TCL='-DUSE_TCL'
+-	TCL_LIB="-ltcl${TCL_VER}"
+-	if [ -e /usr/include/tcl${TCL_VER} ]
+-	then
+-		TCL_INC="-I/usr/include/tcl${TCL_VER}"
+-	elif [ -e /usr/include/tcl.h ]
+-	then
+-		TCL_INC=""
+-	elif [ -e /usr/local/include/tcl${TCL_VER} ]
+-	then
+-		TCL_INC="-I/usr/local/include/tcl${TCL_VER}"
+-	else
+-		USE_TCL=""
+-		TCL_LIB=""
+-		echo "==> WARNING: no Tcl header files found!"
+-	fi
+-fi
+-if [ -n $USE_TCL ]
+-then
+-	LIBPOSTFIX=`ls -1 /usr/local/lib/ /usr/lib | grep 'libtcl[0-9]' | grep so | sed -e 's/\.so.*//g' -e 's/libtcl//g' | sort -r | head -1`
+-	TCL_LIB="-ltcl${LIBPOSTFIX} -lm -lpthread"
+-fi
++#for TCLPATH_TRY in "/data/data/com.termux/files/usr/bin/"
++#do
++#	for TCLVER_TRY in "8.6" "8.3" "8.2" "8.1" "8.0"
++#	do
++	#	if [ -z $TCLSH ]
++	#	then
++		#	TCLSH_TRY=${TCLPATH_TRY}tclsh${TCLVER_TRY}
++		#	if [ -f $TCLSH_TRY ]
++			#then
++			#	TCLSH=$TCLSH_TRY
++			#	echo "===> Found Tclsh in: $TCLSH"
++			#fi
++		#fi
++	#done
++#done
++#if [ -f $TCLSH ]
++#then
++	#TCL_VER=`echo puts \\$tcl_version | $TCLSH -`
++	#USE_TCL='-DUSE_TCL'
++#	TCL_LIB="-ltcl${TCL_VER}"
++	#if [ -e $PREFIX/include/tcl${TCL_VER} ]
++	#then
++	#	TCL_INC="-I$PREFIX/include/tcl${TCL_VER}"
++	#elif [ -e $PREFIX/include/tcl.h ]
++	#then
++		#TCL_INC=""
++	#elif [ -e $PREFIX/include/tcl${TCL_VER} ]
++	#then
++		#TCL_INC="-I$PREFIX/include/tcl${TCL_VER}"
++	#else
++		#USE_TCL=""
++		#TCL_LIB=""
++		#echo "==> WARNING: no Tcl header files found!"
++	#fi
++#fi
++#if [ -n $USE_TCL ]
++#then
++	#LIBPOSTFIX=`ls -1 $PREFIX/lib/ $PREFIX/lib | grep 'libtcl[0-9]' | grep so | sed -e 's/\.so.*//g' -e 's/libtcl//g' | sort -r | head -1`
++	#TCL_LIB="-ltcl${LIBPOSTFIX} -lm -lpthread -landroid-shmem"
++#fi
```

Comment:
If none of this is used/needed then it can probably be removed inatead of commented.